docs/BR.md

```diff
@@ -1756,142 +1754,916 @@ The CA SHALL enforce multi-factor authentication for all accounts capable of dir
+
+| __Field__                      | __Description__ |
+| ---                            | ------          |
+| `tbsCertificate`               | |
+| \ \ \ \ `version`              | MUST be v3(2) |
+| \ \ \ \ `serialNumber`         | MUST be a non-sequential number greater than zero (0) containing at least 64 bits of output from a CSPRNG. |
+| \ \ \ \ `signature`            | See [Section 7.1.3.2](#7132-signature-algorithmidentifier) |
+| \ \ \ \ `issuer`               | MUST be byte-for-byte identical to the `subject` field of the Issuing CA. See [Section 7.1.4.1](#7141-name-encoding) |
+| \ \ \ \ `validity`             | __**TBD**__ |
+| \ \ \ \ `subject`              | See [Section 7.1.2.8.1](#71281-ca-certificate-naming) |
+| \ \ \ \ `subjectPublicKeyInfo` | See [Section 7.1.3.1](#7131-subjectpublickeyinfo) |
+| \ \ \ \ `issuerUniqueID`       | MUST NOT be present |
+| \ \ \ \ `subjectUniqueID`      | MUST NOT be present |
+| \ \ \ \ `extensions`           | See [Section 7.1.2.3.1](#71231-technically-constrained-non-tls-subordinate-ca-extensions) |
+| `signatureAlgorithm`           | Encoded value MUST be byte-for-byte identical to the `tbsCertificate.signature`. |
+| `signature`                    | |
+
+##### 7.1.2.3.1 Technically Constrained Non-TLS Subordinate CA Extensions
+
+| __Extension__                     | __Presence__  | __Critical__          | __Description__ |
+| ----                              | -             | -                     | ----- |
+| `authorityKeyIdentifier`          | MUST          | N                     | See [Section 7.1.2.9.1](#71291-authority-key-identifier) |
+| `basicConstraints`                | MUST          | Y                     | See [Section 7.1.2.8.3](#71283-basic-constraints) |
+| `certificatePolicies`             | MUST          | N                     | See [Section 7.1.2.8.4](#71284-certificate-policies---affiliated-ca) (Affiliated CA) |
+| `crlDistributionPoints`           | MUST          | N                     | See [Section 7.1.2.9.2](#71292-crl-distribution-points) |
+| `keyUsage`                        | MUST          | Y                     | See [Section 7.1.2.8.8](#71288-key-usage) |
+| `subjectKeyIdentifier`            | MUST          | N                     | See [Section 7.1.2.9.4](#71294-subject-key-identifier) |
+| `extKeyUsage`                     | MUST[^eku_ca] | N                     | See [Section 7.1.2.8.5](#71286-extended-key-usage---non-tls-ca) (Non-TLS CA) |
+| `authorityInformationAccess`      | SHOULD        | N                     | See [Section 7.1.2.8.2](#71282-authority-information-access) |
+| `nameConstraints`                 | MAY           | \*[^name_constraints] | See [Section 7.1.2.8.9](#71289-name-constraints) |
```

Comment:
The name constraints section being pointed to here appears to be BR TLS-centric. A separate table may be needed.

Reply:
> This seems like an overreach of the scope of this work.

I'm not sure how to productively engage with statements like this. It sounds like you haven't responded in substance to the analysis of our **existing requirements** other than to say "No", and I'm not sure how to best address that. Would you like to share your analysis as to why 7.1.5 doesn't apply from the existing BRs?

> Secondarily, other trust frameworks may allow wildcard characters that violate the LDH syntax to appear.

This either misunderstands or misrepresents RFC 5280. The `nameConstraint` syntax is fixed, you're discussing `subjectAltName` values. Explicitly, `nameConstraints` do not, nor have they ever, allowed wildcards as you describe: the entire point of specifying the nameConstraint rules was as an alternative expression of wildcards. So this argument is not valid.

> but I think the current proposal is too restrictive and potentially creeps into other WGs' scope.

This isn't really accurate, though. This concern is only applicable in the context of having a single "Root" CA subject to multiple independent sets of requirements, and then wanting to issue subordinate CAs that are subject to only one set of requirements. Yet that's functionally impossible to achieve, because if a _CA_ is subject to one set of requirements, _everything_ it issues is also subject to those requirements.

As a concrete example: A SCWG BR prohibition against using SHA-1 with a Root Private Key is already an imposition into a SMCWG use case that says "I want to support SHA-1 because email". There naturally is overlap here, in that each CWG has to define what, definitionally, makes something "out of scope". It sounds like you're arguing that "EKU alone is sufficient to make it out of scope", but that's _not_ the case today (see https://github.com/cabforum/servercert/issues/314 )

As I said, this is an **existing requirement** in 7.1.5, so the only new element here is with respect to uplifting the Mozilla policy requirements as to what constitutes "Technically Constrained" re: `SRVName`. The choice in doing this was to avoid an ambiguity where something is "Technically Constrained by the BRs" but not "Technically Constrained"; if we want to give that up, sure, but then I think it'd be all the more reason to be harder on any CA that misunderstands these requirements.

Reply:
We should take this to the list, because I can't help but feel you're quite confused about the standards, and this sort of confusion is quite dangerous for CAs.

This discussion also seems to try to sling as many ideas as possible to get them to stick, and thus can make it difficult to address each of them meaningfully.

1. You've not addressed whether or not you believe the existing 7.1.5 applies to certificates issued by CAs subject to the BRs. This has been raised several times now, and is somewhat core, because all of your subsequent arguments are irrelevant if this is the case. If you believe it's not the case, you do have a burden to demonstrate this.
2. It would appear that you're suggesting that RFC 5280 is ambiguous, even though this language, for 22 years and myriad implementations, has never been seen as such. In particular, it appears you're suggesting that the nameConstraint field supports arbitrary characters (i.e. beyond the syntax of subjectAltName). This, despite the contextual discussion of the time, both within the IETF and the broadly held understanding of a "host name" being that in the preferred name syntax as specified by the relevant RFCs, and which has been repeatedly reaffirmed through related RFCs (e.g. RFC8449, aka BCP 219).
3. It would appear that you're suggesting that, in the context of nameConstraints, wildcard characters are thus implicitly permitted, even though never explicitly as such, and that, when present in a nameConstraint, can have an interpretation that extends the syntax or meaning of a nameConstraint beyond that specified in RFC 5280. This is a very serious claim, and one which would have profound security implications if true, as it would suggest that different implementations could reach different conclusions with respect to the interpretation of nameConstraints. For example, does "`foo*.example.com`" as a nameConstraint somehow imply that a CA is constrained to permit "`foobar.example.com`" but not "`baz.example.com`"? Your explanation would suggest that "it depends", which is antithetical to the goal and purpose of RFC 5280, and for which no RFC exists that describes such interaction with respect to nameConstraints. Thus, this objection seems without any objective merit based in implementation reality or widely held belief, but that you persist in arguing for it suggests you believe it deeply to be essential. This bears further discussion.
4. It would appear that you're suggesting that we should ignore the *current* BRs definition of technical constraints, with respect to **technically-constrained CAs**, or to redefine it such that CAs are allowed permission to sign arbitrary things in some cases. This is equally a quite dangerous interpretation, and bears discussion.

Look, I can understand wanting to highlight concerns, but I do believe that this concern lacks _substance_ or _merit_. It's unclear if you're trying to make a point on what you believe is a technicality, despite it not being supported by the existing requirements, or whether you believe there is a legitimate use case being unaddressed.

In any discussion, I hope you'd address the core of the concern, which is what the **current** requirements state, with respect to the requirements on nameConstraints in certificates issued by CAs subject to the BRs, even those that lack the serverAuth EKU. I do not believe the **current** BRs support your interpretation, and that moots the entirity of your argument.

However, should we reach a conclusion that the **current** BRs fail to account for this, despite evidence to the contrary, then it equally appears you're taking an interpretation that is unsupported by any of the relevant information, including IETF BCPs. I hope you can demonstrate why with something more meaningful and substantive as to a concrete use case, because it does not seem worthwhile to waste time redefining nameConstraints in a way contrary to two decades worth of implementation experience without substantial justification.

Reply:
Resolving this with https://github.com/sleevi/cabforum-docs/pull/36/commits/45f634bfd8c1d8c9d0b7544aabf423820fd1ac11 and https://github.com/sleevi/cabforum-docs/pull/36/commits/90d16e177b1b0ef5bbbaccaf9470757132b08919 and https://github.com/sleevi/cabforum-docs/pull/36/commits/74c89be1716a1d1c70e2fc98a7f7878d5dd65f91

However, do comment if we should re-open, or we can take to the list (since I think this may overlap some with the related discussion about subject names containing Domain Names, and what constitutes misleading/unvalidated information, etc)